### PR TITLE
Issue/278/fast transient module

### LINKED ISSUE #278: [Fast Transient] Rate module
Module implementing the rate computation for detection of fast evolving transient

see : [Fast_Transient_workpaper](https://docs.google.com/document/d/1n9ju5YCqo1vQGGZP6TKBuP2jQ5MGcP2i/edit#)

### PR DESCRIPTION
**Pull request by FusRoman**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): close #278 

## If this is a new release, did you issue the corresponding schema in fink-client?

_otherwise you'll break the livestream service!_

## What changes were proposed in this pull request?

How is the issue this PR is referenced against solved with this PR?

### CODE REVIEW

**Review by dturpin-svom — APPROVED** [review]

File: fink_science/fast_transient_rate/utils.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import numpy as np
+
+
+def to_flux(mag):
+    # from Serguey Karpov
```
Comment (JulienPeloton) [inline fink_science/fast_transient_rate/utils.py:5]: `Sergey`

File: fink_science/fast_transient_rate/processor.py [hunk 2]
```diff
@@ -0,0 +1,409 @@
+import numpy as np
```
Comment (JulienPeloton) [inline fink_science/fast_transient_rate/processor.py:1]: Header missing

File: fink_science/fast_transient_rate/__init__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+from pyspark.sql.types import DoubleType, BooleanType
```
Comment (JulienPeloton) [inline fink_science/fast_transient_rate/__init__.py:1]: header missing

File: fink_science/fast_transient_rate/utils.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import numpy as np
```
Comment (JulienPeloton) [inline fink_science/fast_transient_rate/utils.py:1]: Header missing

**Review by JulienPeloton — APPROVED** [review]
Thanks @FusRoman -- it looks good to me! I left some minor comments, but then it can be merged.

**Pull request merged**